numpy/core/tests/test_function_base.py

```diff
@@ -402,3 +402,8 @@ def test_object(self):
         stop = array(2, dtype='O')
         y = linspace(start, stop, 3)
         assert_array_equal(y, array([1., 1.5, 2.]))
+
+    def test_round_negative(self):
+        y = linspace(-5, 5, num=10, endpoint=False, dtype=int)
+        t = array([-5, -4, -3, -2, -1, -0, 1, 2, 3, 5], dtype=int)
```

Comment:
```suggestion
        t = array([-5, -4, -3, -2, -1, 0, 1, 2, 3, 5], dtype=int)
```
Int zero is not signed.

What does this test give before your change?